frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/create-obc.tsx

```diff
@@ -0,0 +1,125 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { Link } from 'react-router-dom';
+import { ActionGroup, Button } from '@patternfly/react-core';
+import { k8sCreate, K8sResourceKind, referenceFor } from '@console/internal/module/k8s';
+import { ButtonBar, history, resourceObjPath } from '@console/internal/components/utils';
+import { StorageClassDropdown } from '@console/internal/components/utils/storage-class-dropdown';
+import { NooBaaObjectBucketClaimModel } from '@console/noobaa-storage-plugin/src/models';
+import { match } from 'react-router';
+import { commonReducer, defaultState } from '../object-bucket-page/state';
+
+export const CreateOBCPage: React.FC<CreateOBCPageProps> = (props) => {
+  const [state, dispatch] = React.useReducer(commonReducer, defaultState);
+
+  const namespace = _.get(props, 'match.params.ns');
+
+  React.useEffect(() => {
+    const obj: K8sResourceKind = {
+      apiVersion: `${NooBaaObjectBucketClaimModel.apiGroup}/${
```

Comment:
`apiVersionForModel` can be used. https://github.com/openshift/console/blob/master/frontend/public/module/k8s/k8s.ts#L39